integration-test/helpers/pages.js

```diff
@@ -194,6 +195,22 @@ export function loginPage (page, server, opts = {}) {
             if (platform === 'android') {
                 expect(JSON.parse(sent)).toEqual(expected)
             }
+        },
+        /**
+         * This is here to capture EXISTING functionality of `macOS` in production and prevent
+         * accidental changes to how `showAutofillParent` messages are sent
+         * @returns {Promise<void>}
+         */
+        async assertClickAndFocusMessages () {
+            const calls = await page.evaluate('window.__playwright.mocks.calls')
```

Comment:
I think we should pass in a function here instead rather than a string; this has the advantage of getting TS and ESLint tooling for free.

As you mentioned I think filtering in the child execution context makes some sense too as you're reducing the data sent and also get typing data on the return value too.

Reply:
Thanks @jonathanKingston, agreed :)